### PR TITLE
Add pathtools recipe

### PR DESCRIPTION
**Pull request by goanpeca**

### CODE REVIEW

File: recipes/pathtools/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  {{ hash_type }}: {{ hash }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
```
Comment (jakirkham) [inline recipes/pathtools/meta.yaml]: Needs `setuptools` for the above args to work.

Comment (jakirkham) [inline recipes/pathtools/meta.yaml]: Done below. 😄

Comment (goanpeca) [inline recipes/pathtools/meta.yaml]: :smile_cat:

**Comment by jakirkham** [comment]
Thanks @goanpeca.

**Comment by goanpeca** [comment]
:+1

**Pull request merged**